bin/main.go

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
Did you mean to include these?

Reply:
yeap this is a binary i'm writing in Go to distribute this github action (also a practice for me to write some concurrency in Go)